src/test/java/hw2/ex2/DifElementsPageTest.java

```diff
@@ -0,0 +1,92 @@
+package hw2.ex2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class DifElementsPageTest {
+
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void ChromeSetUp()
+    {
+        driver = new ChromeDriver();
+    }
+
+    @Test (description = "Different Elements Page Test")
+    public void MyTest_02()  {
```

Comment:
Very bad name for the method, not informative.

Reply:
changed to DifferentElementsPageTestMethod